docs/docs/60-new-docs/50-user-guide/20-how-to-guides/16-working-with-warehouses.md

```diff
@@ -0,0 +1,198 @@
+artifact sources. It acts as a central hub for subscribing to and monitoring changes in:
+
+- Container image repositories
+- Git repositories
+- Helm chart repositories
+
+`Warehouse`s are responsible for watching these sources and producing new `Freight` whenever changes are
+detected in the subscribed repositories. Each piece of `Freight` represents a collection of specific
+versions of artifacts that can be promoted through your deployment pipeline.
+
+:::info
+To see how `Warehouse` fits into the overall architecture,
+refer to the [Warehouse section of the Core Concepts doc](./../10-core-concepts/index.md#warehouses).
+:::
+
+## The `Warehouse` Resource Type
+
+A `Warehouse` resource's most important field is its `spec.subscriptions` field,
+which is used to subscribe to one or more artifact sources.
+
+Here's an example of a `Warehouse` subscribing to both a
+container image repository and a Git repository:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Warehouse
+metadata:
+  name: my-warehouse
+  namespace: kargo-demo
+spec:
+  subscriptions:
+  - image:
+      repoURL: public.ecr.aws/nginx/nginx
+      semverConstraint: ^1.26.0
+  - git:
+      repoURL: https://github.com/example/kargo-demo.git
+```
+
+:::info
+Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
+:::
+
+### Image Subscription
+
+For subscriptions to container image repositories, the `imageSelectionStrategy` field specifies the method for selecting
+the desired image. The available strategies for subscribing to an image repository are:
+
+- `Digest`: This strategy is used when subscribing to a specific mutable tag, such as `latest`, which is generally
+    discouraged due to best practices favoring immutable tags. Users must supply a value in the `constraint` field,
+    specifying the mutable tag they wish to track. The strategy will retrieve the latest details for the image
+    tagged in this way, including any new or updated digest.
+
+- `Lexical`: This strategy selects the image with the lexicographically greatest tag, making it suitable
+    for scenarios where tags incorporate date/time stamps in formats like `yyyymmdd`. When using this
+    strategy, it's recommended to pair it with a regular expression in the `allowTags` field to limit
+    eligibility to tags that match the expected format, ensuring the correct selection of tags.
+
+- `NewestBuild`: This strategy selects the image with the most recent build time.
+
+    :::warning
+    `NewestBuild` requires retrieving metadata for every eligible tag, which can be slow and is likely to
+    exceed the registry's rate limits. It's advisable to use the `allowTags` field to limit
+    the number of tags for which metadata is retrieved, thereby reducing the risk of hitting rate limits.
+    :::
+
+- **SemVer**: This strategy selects the image that best matches a semantic versioning constraint.
```

Comment:
```suggestion
- `SemVer`: This strategy selects the image that best matches a semantic versioning constraint.
```

I also recommend making this the first one listed, since it's the default. We should also mention that it's the default.